examples/undocumented/libshogun/splitting_timeseries_crossvalidation.cpp

```diff
@@ -0,0 +1,100 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
```

Comment:
plz use the BSD license header from here (currently we are in transition to BSD based license): 
https://github.com/shogun-toolbox/shogun/blob/develop/src/shogun/lib/any.h#L2